antarest/storage/repository/filesystem/folder_node.py

```diff
@@ -55,20 +54,29 @@ def save(self, data: JSON, url: Optional[List[str]] = None) -> None:
             (name,), sub_url = self.extract_child(children, url)
             return children[name].save(data, sub_url)
         else:
-            self.validate(data)
```

Comment:
this method should be tranfered in StorageService::import_study and the result stored in the metadata study object